src/spinner.test.aria.ts

```diff
@@ -0,0 +1,10 @@
+import { expect, test } from '@playwright/test';
+
+test('size="medium"', async ({ page }, test) => {
```

Comment:
Not sure what to name this 🤔.

Reply:
Hm. Tricky given our naming convention for aria tests, but nothing else comes to mind without adjusting how we write them 🤔

Reply:
Me either. I'll just leave a code comment.